### ISSUE TITLE
Provide a faster calibration scheme for Raven

### ISSUE THREAD

**Issue by richardarsenault** [Ouranosinc/raven#168]
Ostrich is too slow to calibrate Raven for many model runs and longer time-series. Therefore we will add the Raven-DDS option, which internally performs the DDS calibration inside RAVEN, thus completely eliminating the need for disk I/(o) and speeding-up the process.

**Comment by richardarsenault** [comment]
It would seem that adding the :DontWriteWatershedStorage and :SupressOutputs tags significantly accelerates the calibration speed, which would make this point much less important.

**Comment by richardarsenault** [comment]
@julemai @huard Do we know what is happening with this? Has the DDS-in-Raven idea been implemented? Has there been progress on the slow-down we encounter while calibrating on the server? I seem to recall there was an issue with Finch/Birdy or something to that effect that could be the source of the slow-down for larger calibration runs?

**Comment by julemai** [comment]
Hi! Sorry for the lengthy reply to follow...

The DDS-in-Raven has been implemented but it is not able to calibrate all parameters at this point; only parameters in the RVP file but not in RVT and RVI and RVH (of which we have several in multiple models). Also "derived" parameters (e.g., para x1 needs to be larger than x2; of which we have several in multiple setups as well) are not covered by that method. Means, if we use the DDS-in-Raven, we need to come up with new calibration setups (keeping several parameters fixed that we would calibrate with OSTRICH).

I think we discussed that in a ticket (can't find it) and decided to stick with Ostrich.

The slow down is also likely not caused by Raven and/or Ostrich. We are running those setups on ComputeCanada and our personal machines and don't have any issues even with large budgets. They scale linearly on all those systems. We however had problems on CC with large IOPs. We thought we solved that; especially with the new Raven versions we use in PAVICS. Could this be an issue when we run the calibration with Ostrich on the server?
